samples/apply-renderers-to-scene-layer/src/main/java/com/esri/arcgismaps/sample/applyrendererstoscenelayer/components/ApplyRenderersToSceneLayerViewModel.kt

```diff
@@ -0,0 +1,244 @@
+ *
+ */
+
+package com.esri.arcgismaps.sample.applyrendererstoscenelayer.components
+
+import android.app.Application
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.viewModelScope
+import com.arcgismaps.Color
+import com.arcgismaps.geometry.Point
+import com.arcgismaps.geometry.SpatialReference
+import com.arcgismaps.mapping.ArcGISScene
+import com.arcgismaps.mapping.ArcGISTiledElevationSource
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.Surface
+import com.arcgismaps.mapping.Viewpoint
+import com.arcgismaps.mapping.layers.ArcGISSceneLayer
+import com.arcgismaps.mapping.symbology.ClassBreak
+import com.arcgismaps.mapping.symbology.ClassBreaksRenderer
+import com.arcgismaps.mapping.symbology.ColorMixMode
+import com.arcgismaps.mapping.symbology.MaterialFillSymbolLayer
+import com.arcgismaps.mapping.symbology.MultilayerMeshSymbol
+import com.arcgismaps.mapping.symbology.SimpleRenderer
+import com.arcgismaps.mapping.symbology.SymbolLayerEdges3D
+import com.arcgismaps.mapping.symbology.UniqueValue
+import com.arcgismaps.mapping.symbology.UniqueValueRenderer
+import com.arcgismaps.mapping.view.Camera
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialogViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+
+/**
+ * ViewModel for the ApplyRenderersToSceneLayer sample.
+ * Displays a 3D buildings scene layer and allows switching between different renderers.
+ */
+class ApplyRenderersToSceneLayerViewModel(app: Application) : AndroidViewModel(app) {
+    // URLs for the Helsinki buildings scene and world elevation
+    private val helsinkiBuildingsSceneLayerUrl =
+        "https://services.arcgis.com/V6ZHFr6zdgNZuVG0/arcgis/rest/services/Helsinki_buildings/SceneServer"
+    private val worldElevationServiceUrl =
+        "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+
+    // ArcGISTiledElevationSource for world elevation
+    private val elevationSource: ArcGISTiledElevationSource by lazy {
+        ArcGISTiledElevationSource(worldElevationServiceUrl)
+    }
+
+    // ArcGISSceneLayer for Portland buildings
+    val sceneLayer: ArcGISSceneLayer by lazy {
+        ArcGISSceneLayer(helsinkiBuildingsSceneLayerUrl)
+    }
+
+    // Camera location for Helsinki
+    private val cameraLocation : Point by lazy {
+        Point(
+            x = 2778453.800861455,
+            y = 8436451.388274617,
+            z = 387.45244609192014,
+            spatialReference = SpatialReference.webMercator()
+        )
+    }
+
+    // Camera to view the scene
+    private val camera : Camera by lazy {
+        Camera(
+            locationPoint = cameraLocation,
+            heading = 308.9,
+            pitch = 50.7,
+            roll = 0.0
+        )
+    }
```

Comment:
Doesn't need `lazy`: 
```suggestion
    private val camera = Camera(
        locationPoint = cameraLocation,
        heading = 308.9,
        pitch = 50.7,
        roll = 0.0
    )
```